### ISSUE TITLE
aria-hidden="true" for equations doesn't work

### ISSUE THREAD

**Issue by Stijn-Otten-Delft** [NULLx76/CSHub3#1]
The attribute `aria-hidden="true" ` doesn't seem to work making all the math display twice

**Comment by NULLx76** [comment]
fixed in #cf8507b
